apluslms_roman/backends/docker.py

```diff
@@ -79,17 +81,38 @@ def prepare(self, task, observer):
         for step in task.steps:
             observer.step_preflight(step)
             image, tag = step.img.split(':', 1)
+
+            day = timedelta(days=1)
+            cache = CacheFile.load(allow_missing=True)
+            last_updated = cache.mlget('images.%s' % (step.img,), None)
```

Comment:
Overly using the mlget feels bit complex.First, you could pass tuple `.mlget(('images', step.img), None)` or you could get the images `images = cache.setdefault('images', {})` and then use `last_updated = images.get(step.img, None)`